docs/quickstart_guide.md

```diff
@@ -118,7 +118,10 @@ kn service delete --all
 
 ## Setup a single-node cluster (master and worker functionality on the same node)
 
-```
+```bash
```

Comment:
this cannot be one single code snippet. This must be two subsubsections. The manual one should appear first because it illustrates how things work, followed by the script. Also, it should be explicitly stated that it not only starts but **restarts** as well.